File: web/avito.php
<?php
$py = 'python3 avito.py';

$score = floatval($_POST['score']);
$category_id = intval($_POST['category_id']);
$has_diff_location = isset($_POST['has_diff_location']) ? 1 : 0;
$uncounted = intval($_POST['uncounted']);
$count_procent = floatval($_POST['count_procent'] / 100);
$bad_item = isset($_POST['bad_item']) ? 1 : 0;
$model = $_POST['model'];

$params = array(
    'score' => array($score),
    'category_id' => array($category_id),
    'has_diff_location' => array($has_diff_location),
    'uncounted' => array($uncounted),
    'count_procent' => array($count_procent),
    'bad_item' => array($bad_item),
    'model' => array($model)
);

$paramsJson = json_encode($params);

$command = $py . ' ' . escapeshellarg($paramsJson);

$result = exec($command, $output, $returnCode);

echo json_encode($result);

File: web/avito.py
import pandas as pd
import joblib
import json
import sys
from sklearn.metrics import accuracy_score, log_loss, confusion_matrix
from sklearn.naive_bayes import GaussianNB
from sklearn.ensemble import RandomForestClassifier, GradientBoostingClassifier
from sklearn.model_selection import train_test_split, cross_validate, cross_val_score, cross_val_predict
from sklearn.neural_network import MLPClassifier
from sklearn.linear_model import LogisticRegression, SGDClassifier
from sklearn.tree import DecisionTreeClassifier
from sklearn.svm import SVC, LinearSVC
from sklearn.neighbors import KNeighborsClassifier
from xgboost import XGBClassifier

params_json = sys.argv[1]
params = json.loads(params_json)

clf = joblib.load('avito-'+params['model'][0]+'.pkl')

new_data = pd.DataFrame({'score': [params['score'][0]],
                         'category_id': [params['category_id'][0]],
                         'has_diff_location': [params['has_diff_location'][0]],
                         'uncounted': [params['uncounted'][0]],
                         'count_procent': [params['count_procent'][0]],
                         'bad_item': [params['bad_item'][0]]})

prediction = clf.predict(new_data)
model_name = clf.__class__.__name__

if prediction == 1:
    print(model_name,": Подозрительный профиль")
else:
    print(model_name,": Профиль не вызывает подозрений")